relay-server/src/actors/outcome_aggregator.rs

```diff
@@ -114,92 +204,35 @@ impl OutcomeAggregator {
             )
     }
 
-    fn flush(&mut self, _context: &mut <Self as Actor>::Context) {
+    fn flush(&mut self) {
+        self.flush_handle = Either::Left(future::pending());
```

Comment:
_nit_: To be super defensive, this could be moved into `do_flush`, which ensures that any timeout is reset on every flush. There is virtually no overhead in doing this operation.